source/manual/howto-manually-remove-assets.html.md

```diff
@@ -11,12 +11,9 @@ review_in: 6 months
 If you need to remove an asset manually from `assets.publishing.service.gov.uk`,
 follow these steps:
 
-1. Get a Rails console for Asset Manager
-1. `asset = Asset.find("asset-id-from-url")` (e.g. `57a9c52b40f0b608a700000a`) or for a Whitehall asset `asset = WhitehallAsset.find_by(legacy_url_path: '/government/uploads/system/uploads/attachment_data/file/id/path.extension')`
-1. Make a note of the path of the asset: `asset.file.path` - this will be used to delete the file from the file system
-1. Mark the asset as deleted (and remove it from S3 if necessary - this will prevent restoration)
-   - [`rake assets:delete[<asset.id>]`][rake-delete]
-   - [`rake assets:delete_and_remove_from_s3[<asset.id>]`][rake-delete-and-remove-from-s3]
+1. Mark the asset as deleted and remove it from S3
+   - [`rake assets:delete_and_remove_from_s3[<asset.id>]`][rake-delete-and-remove-from-s3] (non-Whitehall assets)
+   - [`rake assets:whitehall_delete_and_remove_from_s3[<full URL path>]`][whitehall-rake-delete-and-remove-from-s3] (Whitehall assets)
```

Comment:
I'd be tempted to call this `legacy_url_path`, which a lot of devs are already familiar with and which is what is used by this rake task.
"Full" URL path kind of reads to me like "put the entire URL here", which would fail the task.

Reply:
Sure, I've updated to say that.